jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenSourceFilesConfiguration.java

```diff
@@ -60,22 +62,23 @@ private static Path replaceClassExtensionWithJava(Path path) {
 
     // Gets the classes files in the 'classes' output directory. It finds the files that are classes
     // files by matching them against the .java source files. All other files are deemed resources.
-    Files.list(classesOutputDir)
-        .forEach(
-            classesOutputDirFile -> {
-              Path classesSourceDirFile = replaceClassExtensionWithJava(classesOutputDirFile);
-
-              // Resolves the file in the source directory.
-              Path correspondingSourceDirFile =
-                  classesSourceDir.resolve(classesOutputDir.relativize(classesSourceDirFile));
-              if (Files.exists(correspondingSourceDirFile)) {
-                // Adds the file as a classes file since it is in the source directory.
-                classesFiles.add(classesOutputDirFile);
-              } else {
-                // Adds the file as a resource since it is not in the source directory.
-                resourcesFiles.add(classesOutputDirFile);
-              }
-            });
+    try (Stream<Path> classesOutputDirFileStream = Files.list(classesOutputDir)) {
+      classesOutputDirFileStream.forEach(
+          classesOutputDirFile -> {
```

Comment:
Variable names could be cleaned up a bit. We try to avoid abbreviating. Maybe just classFile and sourceFile if that's what these are?

Reply:
Did some renaming to get rid of all occurrences of `Dir` abbreviation.